src/data/webplatform.yaml

```diff
@@ -11,33 +11,36 @@ products:
   'Audio/Video: Recording',
   'CSS Parsing and Computation',
   'CSS Transitions and Animations',
-  'Document Navigation',
   'DOM: Animation',
   'DOM: Bindings (WebIDL)',
   'DOM: Content Processes',
   'DOM: Core & HTML',
   'DOM: CSS Object Model',
   'DOM: Device Interfaces',
+  'DOM: Drag & Drop',
+  'DOM: Editor',
   'DOM: Events',
-  'DOM: IndexedDB',
   'DOM: File',
+  'DOM: Forms',
+  'DOM: HTML Parser',
+  'DOM: Mozilla Extensions',
```

Comment:
I did a quick check on the advance search tab of bugzilla [website](https://bugzilla.mozilla.org/buglist.cgi?resolution=---&query_format=advanced&component=DOM%3A%20Mozilla%20Extensions&product=Core) and I wonder does this component exists?

Reply:
It was just moving it around to be alphabetically sorted, but I guess this component no longer exists (before the bug mentioned there?)

Reply:
Removed it in a separete commit.